src/test/rustdoc/description.rs

```diff
@@ -0,0 +1,22 @@
+#![crate_name = "foo"]
+//! # Description test crate
+//!
+//! This is the contents of the test crate docstring.
+//! It should not show up in the description.
+
+// @has 'foo/index.html' '//meta[@name="description"]/@content' \
+//   'Description test crate'
```

Comment:
The test doesn't match on the exact string, only that the output contains the test string somewhere.
```suggestion
// @has 'foo/index.html' '//meta[@name="description"]/@content' \
//   'Description test crate'
// @!has - '//meta[@name="description"]/@content' 'should not show up'
```